src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -106,7 +106,9 @@ internal Project(Solution solution, ProjectState projectState)
         /// Immutable snapshot of language services from the host environment associated with this project's language.
         /// Use this over <see cref="LanguageServices"/> when possible.
         /// </summary>
+#pragma warning disable CS0618 // Type or member is obsolete. Use Services instead. - This is the implementation of Services.
```

Comment:
So this was needed now because the compiler previously wasn't processing the Obsolete attribute during the source build?

Reply:
Yes the compiler didn't produce obsolete diagnostics when the attribute argument was interpolated string constant. This error was caught in the boostrapping build.